lighthouse-core/gather/gatherers/offline.js

```diff
@@ -59,23 +42,28 @@ class Offline extends Gatherer {
     return driver.sendCommand('Network.emulateNetworkConditions', Offline.config({offline: false}));
   }
 
-  afterPass(options) {
+  beforePass(options) {
     const driver = options.driver;
 
-    // TODO eventually we will want to walk all network
-    // requests that the page initially made and retry them.
-    return Offline
-        .goOffline(driver)
-        .then(_ => driver.evaluateAsync(`(${requestPage.toString()}())`))
-        .then(offlineResponseCode => {
-          this.artifact = offlineResponseCode;
-        })
-        .then(_ => Offline.goOnline(driver))
-        .catch(_ => {
-          this.artifact = {
-            offlineResponseCode: -1
-          };
-        });
+    return driver.gotoURL(options.url, {waitForLoad: true})
```

Comment:
So I *think* that this initial navigation to `options.url` is wasteful, since the `config.json` for this pass specifies `"loadPage": true`. That being said, when I used `node lighthouse-cli <url> --audit-whitelist=works-offline` to just run this one specific audit, I did end up needing to explicitly add in the initial navigation.

It may be because this is currently in a `beforePass` function. I don't think I could move this code to `afterPass`, since by the time that's called, `tracingData` is already set. Is there some other lifecycle function that I could use instead that would allow me to take advantage of `"loadPage": true`?